src/main/java/com/salesforce/dynamodbv2/mt/mappers/metadata/DynamoTableDescriptionImpl.java

```diff
@@ -172,35 +173,53 @@ public CreateTableRequest getCreateTableRequest() {
     @Override
     public String toString() {
         return "DynamoTableDescriptionImpl{" +
-                "tableName='" + tableName + '\'' +
-                ", attributeDefinitions=" + attributeDefinitions +
-                ", primaryKey=" + primaryKey +
-                ", gsiMap=" + gsiMap.values() +
-                ", lsiMap=" + lsiMap.values() +
-                '}';
+            "tableName='" + tableName + '\'' +
+            ", attributeDefinitions=" + attributeDefinitions +
+            ", primaryKey=" + primaryKey +
+            ", gsiMap=" + gsiMap.values() +
+            ", lsiMap=" + lsiMap.values() +
+            '}';
     }
 
     @Override
     public boolean equals(Object o) {
-        if (this == o) return true;
-        if (o == null || getClass() != o.getClass()) return false;
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
 
         DynamoTableDescriptionImpl that = (DynamoTableDescriptionImpl) o;
 
-        if (!tableName.equals(that.tableName)) return false;
-        if (!attributeDefinitions.equals(that.attributeDefinitions)) return false;
-        if (!primaryKey.equals(that.primaryKey)) return false;
-        if (!gsiMap.equals(that.gsiMap)) return false;
-        if (!lsiMap.equals(that.lsiMap)) return false;
-        if (!provisionedThroughput.getReadCapacityUnits().equals(that.provisionedThroughput.getReadCapacityUnits())) return false;
-        if (!provisionedThroughput.getWriteCapacityUnits().equals(that.provisionedThroughput.getWriteCapacityUnits())) return false;
+        if (!tableName.equals(that.tableName)) {
```

Comment:
probably not appropriate for this PR, but heck this would probably look nicer with one "return false;" inside an if with a bunch of "||"s (but leaving line 216 alone)

Reply:
I think I generated this using intellij and just lived with it.  I'm open to changing it.